src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -2039,15 +2037,15 @@ static void M<T>(int? x)
                 context.CompileAssignment("x", "Test(x is int i)", out error, testData);
                 testData.GetMethodData("<>x.<>m0<T>").VerifyIL(
     @"{
-  // Code size       67 (0x43)
+  // Code size       74 (0x4a)
```

Comment:
> // Code size       74 (0x4a) [](start = 1, length = 29)

It looks like IL got worse

Reply:
Already recorded in [https://github.com/dotnet/roslyn/issues/23222](https://github.com/dotnet/roslyn/issues/23222)

---
In reply to: [156566609](https://github.com/dotnet/roslyn/pull/23209#discussion_r156566609) [](ancestors = 156566609)